base/docs/helpdb/Base.jl

```diff
@@ -2603,7 +2603,8 @@ creates a task, and does not run it.
 """
     fld(x, y)
 
-Largest integer less than or equal to `x/y`.
+Largest integer `n` such that `y * n` is less than or equal to `x`. Note that this can be
```

Comment:
From https://github.com/JuliaLang/julia/issues/17415#issuecomment-232730858 it seems that the current implementation is what it is meant to be. The original doc was also not wrong, just misleading.